recipes/plugins_library/plugin_image.py

```diff
@@ -9,13 +9,16 @@ def check_rules_on_label(label: Dict):
     print("Custom method - checking number of bboxes")
 
     counter = 0
+    issues_array = []
     for annotation in label["jsonResponse"]["JOB_0"]["annotations"]:
         if annotation["categories"][0]["name"] == "OBJECT_A":
             counter += 1
 
     if counter <= 1:
-        return []
-    return [f"There are too many BBox ({counter}) - Only 1 BBox of Object A accepted"]
+        return issues_array
+    return issues_array.append(
+        [f"There are too many BBox ({counter}) - Only 1 BBox of Object A accepted"]
+    )
```

Comment:
This doesn't change anything, no ?

Reply:
I had pylance type error if I didn't do that